tools/install/Makefile.skel

```diff
@@ -143,18 +128,18 @@ styletest:
 	@./tests/run_style_test.sh
 
 unittests: $(fruit_driver)
```

Comment:
I guess these rules (`unittests`, `modeltests`, etc...) could also be moved to the new `makefile.tests` and only the rules `all`, `sharedlib` and `clean` can remain the main `Makefile`

Reply:
That's what I'd do, one clean path through to compile the executables. That's what I'd want as a user with no interest in the test cases other than they work in ci.yml